tide/master/hardware/ScreenController.h

```diff
@@ -37,8 +38,8 @@
 /* or implied, of Ecole polytechnique federale de Lausanne.          */
 /*********************************************************************/
 
-#ifndef ScreenController_H
-#define ScreenController_H
+#ifndef SCREEN_CONTROLLER_H
+#define SCREEN_CONTROLLER_H
```

Comment:
I prefer #pragma once

Reply:
I agree that it's less work and less error prone, I was using pragma once before joining the BBP then I stopped to maintain a consistent style, and now I got used to doing it by hand... ;-) The only times I found hard-coded names to be useful is in the rare cases when you need fine control over which files are included when using conflicting or broken third-party libraries. But that's very uncommon and I agree that it does not apply to this project. Maybe I should consider doing a batch change at some point.